frontend/language/src/nb.json

```
@@ -1880,6 +1881,8 @@
   "ux_editor.properties_panel.subform_table_columns.choose_component_description": "Listen viser bare de komponentene som har minst én datamodellbinding og en ledetekst",
   "ux_editor.properties_panel.subform_table_columns.column_cell_content": "Celleinnhold (query): <0>{{cellContent}}</0>",
   "ux_editor.properties_panel.subform_table_columns.column_header": "Kolonne {{columnNumber}}",
+  "ux_editor.properties_panel.subform_table_columns.column_multiple_data_model_bindings_description": "Komponenten inneholder flere felt og kan kun vise et felt per kolonne",
```

Just a small nitpick/typo, but not blocking so feel free to resolve this when changed 👍 
```suggestion
  "ux_editor.properties_panel.subform_table_columns.column_multiple_data_model_bindings_description": "Komponenten inneholder flere felt og kan kun vise ett felt per kolonne",
```